### PR TITLE
[terra-clinical-detail-view] Fixed IE10 wrapping issue for label-value-views

### LINKED ISSUE #516: [clinical-detail-view] IE wrapping issue for label-value-views
# Bug Report
Wrapping issue with `clinical-label-value-view` inside of `detail-view-list` on IE

## Steps to Reproduce
1. Have long content in a detail-view-list item (label-value-view)
2. View on IE.

## Additional Context / Screenshots
MicrosoftTeams-image (1)

## Expected Behavior
`clinical-label-value-view` in `detail-view-list` should wrap, need to check css to make sure correct styles are being applied. Most likely similar issue with flex bug noted [here](https://stackoverflow.com/questions/35111090/text-in-a-flex-container-doesnt-wrap-in-ie11).

## Environment
* IE 10/11

### PR DESCRIPTION
**Pull request by avinashg1994**
### Summary
Fixed IE10 wrapping issue for label-value-views inside of detail-view-list, by adding `max-width: 100%` in DetailListItem css.

**Screenshots:**
_**IE11**_
IE11

_**IE10**_
IE10

### Additional Details
Verified this in IE11, IE10, Chrome and Mozilla

fixes #516 

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-detail-view/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Unreleased
 ### Changed
 * Update wdio specs to use Terra.describeViewports
 
+### Fixed
+* Fixed IE10 wrapping issue for label-value-views inside of detail-view-list.
```
Comment (yuderekyu) [inline packages/terra-clinical-detail-view/CHANGELOG.md]: want to update this entry? probably state we added a max width to the item that resolves wrapping bug in ie 10

**Comment by emilyrohrbough** [comment]
@avinashg1994 can you add screenshots to verify this change in IE10 and IE11.

File: packages/terra-clinical-detail-view/CHANGELOG.md (hunk 1 above)
Comment (avinashg1994) [reply packages/terra-clinical-detail-view/CHANGELOG.md]: Updated the entry in CHANGELOG.md file to be more specific.

**Comment by avinashg1994** [comment]
> @avinashg1994 can you add screenshots to this PR summary to verify this change in IE10 and IE11.

I have added the screenshots for IE10 and IE11

**Pull request merged**